pkgs/applications/version-management/subversion/default.nix

```
@@ -38,7 +38,8 @@ let
       ++ lib.optional httpSupport serf
       ++ lib.optionals pythonBindings [ python3 py3c ]
       ++ lib.optional perlBindings perl
-      ++ lib.optional saslSupport sasl;
+      ++ lib.optional saslSupport sasl
+      ++ lib.optional stdenv.hostPlatform.isDarwin (with darwin.apple_sdk.frameworks; [ CoreServices Security ]);
```

```suggestion
      ++ lib.optional stdenv.hostPlatform.isDarwin [ CoreServices Security ];
```
Please inherit them from top-level.